.github/workflows/pull-request.yaml

```
@@ -28,3 +28,4 @@ jobs:
           python -m pytest tests/unit
         env:
           NODE_URL: https://rpc.ankr.com/eth
+          PAYOUTS_SAFE_ADDRESS: '0x0000000000000000000000000000000000000000'
```

```suggestion
          PAYOUTS_SAFE_ADDRESS: '0x0000000000000000000000000000000000000000'
          NETWORK=mainnet
```
Would this fix the failing tests?